relay-server/src/envelope.rs

```diff
@@ -162,6 +164,7 @@ impl fmt::Display for ItemType {
             Self::ReplayRecording => write!(f, "replay_recording"),
             Self::CheckIn => write!(f, "check_in"),
             Self::Span => write!(f, "span"),
+            Self::OtelSpan => write!(f, "otel_span"),
```

Comment:
Please also add an entry to `from_str` below. Unfortunately the compiler does not catch this.